### PR TITLE
Update kubedns docs

### PR DESCRIPTION
**Pull request by thockin**

### CODE REVIEW

File: cluster/addons/dns/kube2sky/README.md [hunk 1]
```diff
@@ -26,9 +38,11 @@ mutation (insertion or removal of a dns entry) before giving up and crashing.
 
 `--etcd-server`: The etcd server that is being used by skydns.
 
-`--kube-master-url`: URL of kubernetes master. Required if `--kubecfg_file` is not set.
+`--kube-master-url`: URL of kubernetes master. Required if service account
+tokens are not available and `--kubecfg_file` is not set.
```
Comment (zhouhaibing089) [inline cluster/addons/dns/kube2sky/README.md]: `--kubecfg_file` is a typo?

**Comment by bgrant0607** [comment]
It would be useful to have some actual user documentation for DNS in kubernetes.github.io, also.

**Comment by thockin** [comment]
@ArtfulCoder done

@bgrant0607 yes, I just want to fix this because people keep getting it wrong.

**Comment by thockin** [comment]
ping @ArtfulCoder

**Comment by ArtfulCoder** [comment]
@k8s-bot test this

**Comment by thockin** [comment]
@girishkalele can you take this PR over and decide to either discard it or apply a similar change to new docs?  I am going to be away and I don't want it to just rot.

**Pull request closed without merging**